rholang/src/test/scala/coop/rchain/rholang/interpreter/MatchTest.scala

```diff
@@ -182,4 +182,11 @@ class VarMatcherSpec extends FlatSpec with Matchers {
 
     result should be(Some(Map[Int, Par](0 -> EList(Seq(GInt(4), GInt(20))), 1 -> Par())))
   }
+
+  "Matching a list with a remainder" should "capture the remainder." in {
```

Comment:
Please also add a test that checks that the remainder works in a Receipt.

Reply:
But it doesn't work yet.